vmm/src/igvm/igvm_loader.rs

```diff
@@ -245,7 +287,12 @@ pub fn load_igvm(
                 todo!("unsupported IgvmPageDataType");
             }
             IgvmDirectiveHeader::MemoryMap(_info) => {
-                todo!("Not implemented");
+                #[cfg(feature = "sev_snp")]
+                {
+                    let ram_size = memory_manager.lock().unwrap().current_ram;
+                    let memory_map = generate_memory_map(ram_size);
+                    import_parameter(&mut parameter_areas, _info, memory_map.as_bytes())?;
+                }
```

Comment:
Why is the input `_info` is not used?

Reply:
It is only used under the feature flag sev_snp, so it would complain under normal scenarios.

Reply:
Do you need to error out or warn for non sev_snp cases?

Reply:
Good point I will add an error message for non-sev-snp scenario.

Reply:
Done